components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -513,3 +586,164 @@ func getMergerTOMLPath(args []string, infra *AutomteHAInfraDetails, timestamp st
 
 	return tomlFile, nil
 }
+
+//getConfigFromRemoteServer gets the config for remote server using the commands
+func getConfigFromRemoteServer(infra *AutomteHAInfraDetails, remoteType string, config string) (string, error) {
+	sshUser := infra.Outputs.SSHUser.Value
+	sskKeyFile := infra.Outputs.SSHKeyFile.Value
+	sshPort := infra.Outputs.SSHPort.Value
+
+	remoteIP, remoteService := getRemoteType(remoteType, infra)
+	scriptCommands := fmt.Sprintf(config, remoteService)
+	rawOutput, err := ConnectAndExecuteCommandOnRemote(sshUser, sshPort, sskKeyFile, remoteIP, scriptCommands)
+	if err != nil {
+		return "", err
+	}
+	return rawOutput, nil
+}
+
+//getDecodedConfig gets the decoded config from the input
+func getDecodedConfig(input string, remoteService string) (interface{}, error) {
+	if remoteService == "postgresql" {
+		var src PostgresqlConfig
+		if _, err := toml.Decode(cleanToml(input), &src); err != nil {
+			return nil, err
+		}
+		return src, nil
+	}
+	if remoteService == "opensearch" {
+		var src OpensearchConfig
+		if _, err := toml.Decode(cleanToml(input), &src); err != nil {
+			return nil, err
+		}
+		return src, nil
+	}
+
+	return nil, nil
+}
+
+//getConfigForArgsOpenSearch gets the requested config from the args provided for opensearch
+func getConfigForArgsOpenSearch(args []string, remoteService string) (OpensearchConfig, error) {
```

Comment:
you created a method by combining both open search and Postgres, but not removed the OpenSearch method and its usage